app/views/profile/wall.html.erb

```diff
@@ -1,48 +1,45 @@
 <%= render "shared/tutorial"  %>
 <!-- Tabs navs -->
-<ul class="nav nav-tabs border-0 nav-fill mb-3" id="ex1" role="tablist">
+<ul class="nav nav-tabs border-0 nav-fill mb-3 flex-wrap" id="ex1" role="tablist">
   <li class="nav-item" role="presentation">
-    <%= link_to "Feed", feed_path(uuid: params[:uuid]), class: "nav-link h4 font-weight-bold border-dark  border-top-0 border-left-0 border-right-0 pb-3 border-dark", id: "ex2-tab-1", data: {mdb_toggle: "tab"} %>
+    <%= link_to "Feed", feed_path(uuid: params[:uuid]),
+                class: "nav-link h4 font-weight-bold border-dark border-top-0 border-left-0 border-right-0 pb-3",
+                id: "ex2-tab-1", data: {mdb_toggle: "tab"} %>
   </li>
   <li class="nav-item" role="presentation">
-    <a
-      class="nav-link active border-bottom-0 border-right-0 border-dark"
-      id="ex2-tab-2"
-      data-mdb-toggle="tab"
-      href="#ex2-tabs-1"
-      role="tab"
-      aria-controls="ex2-tabs-2"
-      aria-selected="false"
-    ><h4 class="font-weight-bold">Wall</h4>
+    <a class="nav-link active border-bottom-0 border-right-0 border-dark"
+       id="ex2-tab-2" data-mdb-toggle="tab" href="#ex2-tabs-1"
+       role="tab" aria-controls="ex2-tabs-2" aria-selected="false">
+      <h4 class="font-weight-bold">Wall</h4>
```

Comment:
please convert to ruby if possible. Please state reason if this is intentional, in that case you can add a comment on top of this so ew remember